test/AutoRest.TestServer.Tests/ModelReaderWriterValidation/BaseModelTests.cs

```diff
@@ -8,54 +8,55 @@
 
 namespace AutoRest.TestServer.Tests
 {
-    internal class BaseModelTests : ModelJsonTests<BaseModel>
-    {
-        protected override BaseModel GetModelInstance()
-        {
-            var typeToActivate = typeof(BaseModel).Assembly.GetTypes().FirstOrDefault(t => t.Name == $"Unknown{typeof(BaseModel).Name}") ?? throw new InvalidOperationException("Unable to find BaseModel type");
-            return Activator.CreateInstance(typeToActivate, true) as BaseModel ?? throw new InvalidOperationException($"Unable to create model instance of BaseModel");
-        }
+    // TODO -- uncomment when https://github.com/Azure/autorest.csharp/issues/4221 is resolved
```

Comment:
we have to remove this entire test case because it needs the fix in [this PR](https://github.com/Azure/autorest.csharp/pull/4232) to work properly.
We will enable these test cases and add more